packages/analytics-js-plugins/src/errorReporting/index.ts

```diff
@@ -56,15 +57,15 @@ const ErrorReporting = (): ExtensionPlugin => ({
 
       // attach breadcrumbs
       if (state.reporting.breadCrumbs.value.length > 0) {
-        errorPayload.breadcrumbs = state.reporting.breadCrumbs.value;
+        errorPayload.breadcrumbs = clone(state.reporting.breadCrumbs.value);
       }
 
       // enrich error payload
       const enhancedError = enhanceErrorEvent(errorPayload, errorState, state);
 
       // send it to metrics service
       httpClient.getAsyncData({
-        url: `https://sdk-metrics.dev-rudder.rudderlabs.com/sdkmetrics`, // metrics endpoint: dev
+        url: `https://sdk-metrics.rudderstack.com/sdkmetrics`,
```

Comment:
Let's define it as a constant.

Reply:
we don't need it once terraform changes are done.

Reply:
Will resolve before we merge this PR.

Reply:
Addressed